app/javascript/app/components/ghg-emissions/ghg-emissions.js

```diff
@@ -128,24 +130,37 @@ class GhgEmissionsContainer extends PureComponent {
 
   handleBreakByChange = breakBy => {
     const params = [
-      { name: 'breakBy', value: breakBy.value },
-      { name: 'version', value: '' },
-      { name: 'filter', value: '' }
+      { name: 'source', value: this.props.sourceSelected.value },
+      { name: 'breakBy', value: breakBy.value }
     ];
-    this.updateUrlParam(params);
+    this.updateUrlParam(params, true);
```

Comment:
We lose the IPCC version when we change the break by. Not sure if its important. I think you do it because you want to get rid of all the countries for example

Reply:
Additionally I have just pushed a fix for the filter also.